README.md

```diff
@@ -41,12 +41,36 @@ api.get_sys_dns
       "search"=>["domain.com"]}
 
 # Or you can manually do it:
-api.get('/mgmt/tm/sys/dns')
+api.get(route: '/mgmt/tm/sys/dns')
```

Comment:
Is this the breaking change? I don't want to introduce breaking changes unless it's required for new features.

Reply:
I've kept this portion passive, but have introduced the requirement of passing a `body` or `headers` keyword argument when one should be provided (if one or both of these should not be provided with a request, there are default values which will allow them not to be passed). This is non-passive, but completely required to support this new functionality.